src/Server/modules/Analytics.js

```diff
@@ -0,0 +1,38 @@
+const readCSVModule = require('../modules/ReadCSV.js');
+
+function GetGenderAnalytics(country, typeOfData) {
+    var allCases = readCSVModule.LoadAllCases();
+
+    var femaleNum = 0;
+    var maleNum = 0;
+
+    for (var i = 0; i < allCases.length; ++i) {
+        if (allCases[country] == country) {
+            if (allCases[i].gender == "female") {
+                if (typeOfData == 2 && allCases[i].dead) femaleNum++; // dead
+                else if (typeOfData == 3 && allCases[i].recovered) femaleNum++; //recovered
+                else if (typeOfData == 1) femaleNum++; // confirmed
+            }
+            else if (allCases[i].gender == "male") {
+                if (typeOfData == 2 && allCases[i].dead) maleNum++; // dead
+                else if (typeOfData == 3 && allCases[i].recovered) maleNum++; //recovered
+                else if (typeOfData == 1) maleNum++; // confirmed
+            }
+        }
+    }
+
+    if (femaleNum == 0 && maleNum == 0) return undefined;
+
+    return new GenderAnalytics(femaleNum, maleNum);
```

Comment:
I don't think you need to create a class for your return value. You can return an object likes this
`return {
      femaleNumber : femaleNum,
      maleNumber: maleNum
}`
Up to you if you want to change your code. If you want to leave it like this, let me know and I will merge your pull request

Reply:
Thank you for the suggestion! Fixed it :D